misc. minor documentation changes

xenophenes opened the pull request.

xenophenes commented on docs/operator-docs.asciidoc:
```diff
@@ -806,28 +804,16 @@ You can view the backup:
-pgo show backup mycluster --show-pvc=true
-....
+View the PVC folder and the backups contained therein:
 
```
If you see the commit message related to this, --show-pvc is no longer valid as confirmed by myself and Pramodh. I changed the code beneath to have alternative commands that show the backup paths.

abrightwell replied:
Sounds good. Can you amend your commit message to include the actual commit ID that removed this feature?

xenophenes replied:
Yep, updated!

xenophenes commented on docs/operator-docs.asciidoc:
```diff
@@ -806,28 +804,16 @@ You can view the backup:
-├── mycluster-backups/2017-03-27-13-54-33
-├── mycluster-backups/2017-03-27-13-56-49
-└── mycluster-backups/2017-03-27-14-02-38
-....
```
The command is no longer valid.

xenophenes commented on docs/operator-docs.asciidoc:
```diff
@@ -14,14 +14,14 @@ from source, you can download them from the following:
 cluster and also set up your local user environment with the
 *pgo* client in about 10 seconds.  More details on the Quickstart
-is at the end of this document <<Quickstart>>.
+script is located in the <<Quickstart>> section in this document.
```
That seems to be a link format; it is "linked" in that manner in the current version of the document and it takes you down to the Quickstart section.

xenophenes commented on docs/operator-docs.asciidoc:
```diff
@@ -14,14 +14,14 @@ from source, you can download them from the following:
-script useful https://github.com/CrunchyData/postgres-operator/blob/master/examples/quickstart.sh
+For most users of the Operator who simply want to deploy
+it on a running Kubernetes cluster will find the Quickstart
+script useful: https://github.com/CrunchyData/postgres-operator/blob/master/examples/quickstart.sh
```
It's automatically converted into a link, if you view the operator-docs document.

abrightwell replied:
That's not what I mean. What I mean is something less verbose. What I mean is, `link:<target>[text]`. See example in documentation above this instance.

xenophenes replied:
Updated!

xenophenes commented:
Latest commit has the following updates:

- Added a step to source your .bashrc file, similar to what's in crunchy-containers
- Added a section documenting the various username/password combinations created by default, and where they're created from (in the event the user wants to change these values or know how to authenticate). Previously, no mention was made of the passwords configured or the default accounts.
- Changed wording minorly in the Quickstart section to make it more clear the quickstart script as-is will not work on a standard Kubernetes deployment; it will need to be modified before it will work.
